src/sparseml/transformers/export.py

```diff
@@ -91,12 +91,14 @@
 __all__ = ["export_transformer_to_onnx", "load_task_model"]
 
 MODEL_ONNX_NAME = "model.onnx"
-DEPLOYMENT_FILES: List[str] = [
+EXTERNAL_ONNX_DATA_NAME = "model.data"
```

Comment:
I wonder whether we should dump all the information into a single `data` file, or maybe have multiple files, one per parameter - this is what torch.onnx.save(...) does. However, given that we want to minimize the number of files per stub in sparsezoo, sticking to one file is probably the way to go. This is also how `optimum` does this: they give you onnx model and the respective single data file.